okhttp/src/main/java/okhttp3/internal/ws/WebSocketWriter.kt

```diff
@@ -140,120 +132,51 @@ internal class WebSocketWriter(
     sink.flush()
   }
 
-  /**
-   * Stream a message payload as a series of frames. This allows control frames to be interleaved
-   * between parts of the message.
-   */
```

Comment:
Based on this comment, with this PR merged, [control frames](https://tools.ietf.org/html/rfc6455#section-5.5.1) (CLOSE, PING, PONG) will have to wait till a large message (eg 100..200+ Kb) is fully sent. Right? 

Or did it behave the same before, and the removed code is dead anyway?

Reply:
It's dead code. The input is a `ByteString` which when written to a `Buffer` was written as a single, large message.

It had the behavior of the `serverLargeBufferedPayloadWrittenAsOneFrame` test.